requirements/edx/github.txt

```diff
@@ -56,9 +56,11 @@ git+https://github.com/edx/edx-lint.git@ed8c8d2a0267d4d42f43642d193e25f8bd575d9b
 -e git+https://github.com/edx/edx-reverification-block.git@1e8f5a7fd589951a90bd31a0824a2c01ac9598ce#egg=edx-reverification-block
 git+https://github.com/edx/ecommerce-api-client.git@1.1.0#egg=ecommerce-api-client==1.1.0
 -e git+https://github.com/edx/edx-user-state-client.git@30c0ad4b9f57f8d48d6943eb585ec8a9205f4469#egg=edx-user-state-client
--e git+https://github.com/edx/edx-proctoring.git@release-2015-07-29#egg=edx-proctoring
 -e git+https://github.com/edx/edx-organizations.git@release-2015-08-03#egg=edx-organizations
 
+# when working in devstack comment out edx-proctoring. It's best to not have this in requirements while doing localdev work
```

Comment:
That's more an a comment to the team. We have problems when we are developing on devstack and have uncommitted edits/pushes to origin as if requirements run, it blows away our work. I'll remove this comment.